### ISSUE TITLE
crash when travelling to Lost Cities dimension

### ISSUE THREAD

**Issue by Zygus42** [McJty/LostCities#90]
lostcities-1.12-2.0.0
forge-1.12.2-14.23.1.2607-installer-win

2 diamond blocks under bed surrounded by skulls and crash report:
[crash-2018-02-07_14.36.12-server.txt](https://github.com/McJty/LostCities/files/1702004/crash-2018-02-07_14.36.12-server.txt)

This is a new modpack i'm creating, so no Lost Cities config existed before adding this mod, and I changed no settings in the config as I assumed that it would use default settings.

**Comment by McJty** [comment]
I will check it out. Is this single player or server?

**Comment by McJty** [comment]
This could be caused by another mod adding an invalid biome. I'll add a test to print out which biome does this.

**Comment by Zygus42** [comment]
Singleplayer, and I'm using a new biome generator, so could well be that.

This is a test on build with nothing but the terrain generator mods and Lost Cities - the lack of other mods might make it easier:
[crash-2018-02-08_18.36.16-server.txt](https://github.com/McJty/LostCities/files/1706632/crash-2018-02-08_18.36.16-server.txt)

One odd thing is that a few times before it crashed it did deliver me to a new world, in a cave and then crashed.

I did a vanilla build with no other mods and tried again - worked fine this time, except that when i made another bed to come back to the Overworld, it created half a bed floating in mid air.  I'm wondering if that could be to do with height differences.

**Comment by McJty** [comment]
Did you modify the lost city configs in any way?

**Comment by Zygus42** [comment]
No, haven't touched them yet as I wanted to see the changes in the latest versions before I played with them.

**Comment by McJty** [comment]
Hmm, that last crash you posted seems to indicate there are still other mods present. More specifically:

	| UCHIJAAAA | otgcore              | 1.12         | minecraft.jar                        | None                                     |
	| UCHIJAAAA | openterraingenerator | v4           | OpenTerrainGenerator-1.12.2 - v4.jar | e9f7847a78c5342af5b0a9e04e5abc0b554d69e0 |
	| UCHIJAAAA | biomebundle          | 5.1          | Biome_Bundle-1.12.2-v6.1.jar         | None                                     |

And this is very relevant as those are biome mods

**Comment by McJty** [comment]
So can you try to remove those too and see if that helps?

**Comment by Zygus42** [comment]
Those are all the mods that make up the world generator - they're all designed to work together.  Removing them leaves me at vanilla minecraft with just LostCities added, and that does work, as I've already tested it.

**Comment by McJty** [comment]
Ok, then the problem is most likely in one of those mods. This evening I will release a new version of Lost Cities that will hopefully show which mod causes the problem

**Comment by Zygus42** [comment]
Cool, thanks for that - will be sure to give it a try.

**Comment by Zygus42** [comment]
Ok, it would appear that it's probably a bug with OTG not setting up the Biome Registry properly ?

[crash-2018-02-09_21.07.07-server.txt](https://github.com/McJty/LostCities/files/1711079/crash-2018-02-09_21.07.07-server.txt)

I actually thought that you used your own world gen for the Lost Cities dimension.

Did you want me to contact them about this matter instead ?

**Comment by McJty** [comment]
I do use my own worldgen but biomes can be from other mods. So yes, you should report this to OTG

**Comment by Zygus42** [comment]
It was already reported there under the Biome Bundle issues: https://github.com/BiomeBundle/BiomeBundle/issues/76

Sounds like it's not likely to happen anytime soon.  Is there no way to start up a Lost Cities dimension with vanilla worldgen  instead ?  Which is actually what I want for my build - I don't actually want OTG world gen for the Lost Cities dimension, just the Overworld dimension.

**Comment by McJty** [comment]
That ticket is about something else. That ticket is about support for OTG world gen with Lost Cities. But this ticket is about a real bug that they are somehow messing up the biome registry. So unrelated

**Comment by Zygus42** [comment]
Ah, ok, misunderstood that then.
